cogboard-app/src/main/kotlin/com/cognifide/cogboard/config/validation/admins/AdminsValidator.kt

```diff
@@ -1,6 +1,6 @@
 package com.cognifide.cogboard.config.validation.admins
 
-import com.cognifide.cogboard.config.model.Admins
+import com.cognifide.cogboard.config.model.Admin
```

Comment:
Rename `AdminsValidator.kt ` to `AdminValidator.kt ` to be consistent

Reply:
done